### ISSUE TITLE
AvatarMIMELimitService limits mime types for all uploads, but only for the browser-uploader

### ISSUE THREAD

**Issue by kraftner** [wecodemore/wcm-avatar#59]
I'm not sure how that came to be. Maybe [first it was restricted to only the profile page and later was extended to make the multi-uploader](https://github.com/wecodemore/wcm-avatar/blob/58caa432ab5637aabbcc8bd7dfd9a90e5337b0bc/Bootstrap.php#L142) work not considering that it now affects all uploads.

So if we really want to restrict the upload here we need to make sure it only happens for avatar uploads which will be hard for async uploads. This is the same problem we'd have in #49 

Otherwise we just get rid of this and consider it out of scope for the plugin.

**Comment by franz-josef-kaiser** [comment]
Good catch! As we already discussed: This plugin was branched out of a larger plugin. There it was meant to restrict _all_ uploads. And yes, I really want to leave this one in. Photographers uploading 30MB avatars should not be allowed.

**Comment by kraftner** [comment]
Okay but then we really need to find a clean way to limit this to only the avatar upload.

**Comment by kraftner** [comment]
Oh now that I looked into this some more actually it affects all uploads, but only for the Browser Uploader. When using the Multi-Uploader (plupload) it doesn't do anything ever because it is limited to the screen-bases of `'profile', 'user-edit', 'media', 'upload'` which don't include the `async-upload` endpoint for plupload based uploads.
